src/components/CSSmodules/Navigation.css

```diff
@@ -0,0 +1,41 @@
+.nav-bar {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  top: 0;
+}
+
+.magicians {
+  color: #000;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.nav {
+  display: flex;
+  font-weight: 500;
+  font-size: 25px;
+  padding-top: 1.5rem;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin-right: 2rem;
+  gap: 0.5rem;
+}
+
+.logo {
+  background-color: rgba(243, 158, 172, 0.5);
+  border-radius: 16px;
+  padding: 0 0.5rem;
+  margin-left: 1.5rem;
+  animation-duration: 2s;
+  animation-name: slideIn;
+  animation-iteration-count: infinite;
+}
+
+@keyframes slideIn {
+  from {
+    margin-right: 100%;
+  }
+
+  to {
+    margin-left: 0%;
+  }
```

Comment:
- [ ] due to this animation your navigation keeps sliding in over and over instead of once which makes your project unusable. I would advise removing this to make your application usable